### ISSUE TITLE
About pretrained speech content space

### ISSUE THREAD

**Issue by DaddyJin** [Hangz-nju-cuhk/Talking-Face_PC-AVS#15]
Thank you for the great work.
In the Equation 2, you use **F_c^v** and  **F_c^a** to calculate the loss function  **L_c^v2a**. However, in the code "av_model.py", when **mode == 'sync'**, you use function **sync(self, augmented, spectrogram)**to train the speech content space. 
If i am not wrong, in the function, you use  **F_n** of Non-Identity space and  **F_c^a** to calculate the  loss function  **L_c^v2a**.

Does it play the same role as equation 2?
Looking for your reply and best wishes!

**Comment by Hangz-nju-cuhk** [comment]
@DaddyJin it is nice of you to see into the code. I am sorry that this part is not updated yet to the final version. The training procedure is the same as written in the paper. This was an older version where we tried to sync F_n with F^a and once.

The actual 'sync' is the same as `self.compute_sync_loss(V_mouth_feature, A_mouth_feature, G_losses)` as written in L526.

**Comment by DaddyJin** [comment]
Thank you for the reply.
Looking forward to the training code. ^_^
